tasks/internal_reconcile_report_orphan/internal_reconcile_report_orphan.py

```diff
@@ -26,6 +26,13 @@
 PAGE_SIZE = 100
 
 
+with open("schemas/input.json", "r") as raw_schema:
```

Comment:
Consider wrapping the open and compile in a try/except block to log errors. See https://github.com/nasa/cumulus-orca/blob/32ad7a817484ffdfadba6cbd2e14a4105c196209/tasks/post_to_catalog/post_to_catalog.py#L20

Reply:
added